_pages/kriskirov.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: kriskirov
+pageUrl: kriskirov.github.io
+timestamp: TODAYS-24-2017-04-Mo
+---
```

Comment:
@kriskirov You're getting closer - it has to be year first, then month, then day of month. 👍